### ISSUE TITLE
Complete implementation of Transition screen

### ISSUE THREAD

**Issue by BVasquez07** [BVasquez07/iGuide#18]
- Transition screen has been created
- Further improve the details or any small accessibility features you think might be missing 
- Generate corresponding audio files and add them to asset folder (The idea here is that we will play this audio to the user.)
